ota-plus-web/conf/application.conf

```diff
@@ -37,7 +37,7 @@ play.modules.enabled += "play.filters.csrf.CSRFModule"
 oidc {
   namespace = "default"
   namespace = ${?OIDC_NAMESPACE}
-  namespaceProvider = "com.advancedtelematic.auth.oidc.NamespaceFromIdentity"
+  namespaceProvider = "com.advancedtelematic.auth.oidc.NamespaceFromUserProfile"
```

Comment:
Have to check or update ota-community-edition for this.

Reply:
I think we have someone dedicated on that right? I will reach out to him 👍